acados/dense_qp/dense_qp_qpoases.c

```diff
@@ -81,6 +82,7 @@ int dense_qp_qpoases_calculate_memory_size(dense_qp_dims *dims, void *args_)
     int size = sizeof(dense_qp_qpoases_memory);
 
     size += 1 * nvd * nvd * sizeof(double);        // H
+	size += 1 * nvd * nvd * sizeof(double);        // R
```

Comment:
Stupid question: Do you have 8 spaces for tab or is it shown wrong throughout the PR for a reason?

Reply:
It's the PR, I believe.